pom.xml

```diff
@@ -44,7 +44,7 @@
         <version.org.jboss.arquillian>1.1.13.Final</version.org.jboss.arquillian>
         <version.org.jboss.arquillian.container.weld-embedded>2.0.0.Beta5</version.org.jboss.arquillian.container.weld-embedded>
         <version.org.jboss.jandex>2.0.5.Final</version.org.jboss.jandex>
-        <version.org.jboss.logging>2.0.1.Final</version.org.jboss.logging>
+        <version.org.jboss.logging>3.3.2.Final</version.org.jboss.logging>
```

Comment:
Can this be changed to 3.3.1.Final as I think that's the version used in WF 11+ right now

Reply:
@kenfinnigan Did a force push 🔨